tests/acceptance/install_test.sh

```diff
@@ -41,3 +41,19 @@ function test_install_downloads_the_given_version() {
   assert_file_exists "$install_dir"
   assert_equals "$(printf "\e[1m\e[32mbashunit\e[0m - 0.8.0")" "$("$install_dir" --version)"
 }
+
+function test_install_downloads_the_main_version() {
+  mock git "cp -r .. temp_bashunit"
+  local install_dir="./lib/bashunit"
+  local output
+
+  output="$(./install.sh lib main)"
+
+  assert_contains "Downloading non-stable main" "$output"
+  assert_contains "bashunit has been installed in the 'lib' folder" "$output"
+  assert_file_exists "$install_dir"
+  assert_equals "$(printf "\e[1m\e[32mbashunit\e[0m - (non-stable) main")" "$("$install_dir" --version)"
+
+  # TODO: Remove this after merging https://github.com/TypedDevs/bashunit/pull/212
+  unset -f git
+}
```

Comment:
Extra acceptance test with the combination of main and specifying the installation directory.